dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -1525,11 +1572,9 @@ private void wesServiceInfoHelp() {
     private void printWesHelpFooter() {
         out("Global Optional Parameters:");
         out("  --wes-url <WES URL>                 URL where the WES request should be sent, e.g. 'http://localhost:8080/ga4gh/wes/v1'");
-        out("  --wes-auth <auth>                   Authorization credentials for the WES endpoint, e.g. 'Bearer 12345'");
-        out("  --aws                               Flag indicating whether the WES request will be to an AWS endpoint");
-        out("  --aws-access-key <key>              The AWS access key associated with an IAM user/profile.");
-        out("  --aws-secret-key <key>              The AWS secret key associated with an IAM user/profile.");
-        out("  --aws-region <region>               The AWS region the WES endpoint is located in, e.g. 'us-east-1'.");
+        out("  --wes-auth <authType> <authValue>   Authorization credentials for the WES endpoint, e.g. 'Bearer 12345' or 'aws myProfile'");
```

Comment:
The vote is a little unclear from the storytime, but would recommend not allowing bearer via command-line
(Also note discussion around https://app.slack.com/archives/CKREJ1938/p1637341883053900)

Reply:
Brainstorming:
We could allow tokens via the command line, but encourage using a config file via our documentation. This would be good for the users who want the bear-bones approach, even though it isn't best practice. 

Or, If we don't want to allow `--wes-auth bearer <token>` at all, it seems like we could drop the  entire`--wes-auth` parameter, as the only remaining valid option would be `--wes-auth aws <profile>`. Dropping `--wes-auth` would mean all credentials are required to be passed in via the config file (until AWS environment variables are fully supported.)

Both approaches seem reasonable, thoughts?

Reply:
If a user wanted to run workflows on different WES server providers (maybe unrealistic) and script the process it would be a lot more difficult without the --wes-auth parameter?

Reply:
Still feels weird although I recognize the pain of having to keep editing. 
Maybe should just drop it

Reply:
Decided to remove it for the time being: https://github.com/dockstore/dockstore-cli/pull/99/commits/dc431f8ae47cc5f5f4f25bfd369f58453540e630, we can re-add `--wes-auth` down the line if we want.